.vib/alertmanager/goss/vars.yaml

```diff
@@ -0,0 +1,14 @@
+binaries:
+  - alertmanager
+  - amtool
+directories:
+  - paths:
+      - /opt/bitnami/alertmanager
+      - /opt/bitnami/alertmanager/conf
```

Comment:
By checking `/opt/bitnami/alertmanager/conf/config.yml` below, you are implicitly checking these paths exist, so I'd remove this.